code-scanning/properties/rust-clippy.properties.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "rust-clippy",
+    "creator": "Rust",
```

Comment:
@yongyan-gh please provide a user or a team handle who we can contact to review future PRs on this starter workflow. They should have commitment to reply to such requests within a reasonable time.

Reply:
Hi @bishal-pdMSFT , can you add the group [sarif-sdk-admin](https://github.com/orgs/microsoft/teams/sarif-sdk-admin)

Thanks